### ISSUE TITLE
Sample code fails: ImportError: cannot import name 'EntityId' from 'tb_rest_client.models.models_ce'

### ISSUE THREAD

**Issue by thunfischbrot** [thingsboard/thingsboard-python-rest-client#30]
Running `pip3 install tb-rest-client`, then running the [sample applications](https://github.com/thingsboard/thingsboard-python-rest-client/blob/master/examples/example_application.py) after adapting the REST API URL and Tenant Administrator credentials gives the following error message:

```
user@machine:~/thb-demo# /usr/bin/python3 ./thb-demo.py 
Traceback (most recent call last):
  File "./thb-demo.py", line 5, in <module>
    from tb_rest_client.rest_client_ce import *
  File "/usr/local/lib/python3.7/dist-packages/tb_rest_client/rest_client_ce.py", line 17, in <module>
    from tb_rest_client.rest_client_base import *
  File "/usr/local/lib/python3.7/dist-packages/tb_rest_client/rest_client_base.py", line 24, in <module>
    from tb_rest_client.api.api_ce import *
  File "/usr/local/lib/python3.7/dist-packages/tb_rest_client/api/api_ce/__init__.py", line 1, in <module>
    from .o_auth_2_config_template_controller_api import OAuth2ConfigTemplateControllerApi
  File "/usr/local/lib/python3.7/dist-packages/tb_rest_client/api/api_ce/o_auth_2_config_template_controller_api.py", line 20, in <module>
    from tb_rest_client.api_client import ApiClient
  File "/usr/local/lib/python3.7/dist-packages/tb_rest_client/api_client.py", line 32, in <module>
    import tb_rest_client.models.models_ce
  File "/usr/local/lib/python3.7/dist-packages/tb_rest_client/models/models_ce/__init__.py", line 3, in <module>
    from .admin_settings_id import AdminSettingsId
  File "/usr/local/lib/python3.7/dist-packages/tb_rest_client/models/models_ce/admin_settings_id.py", line 17, in <module>
    from . import EntityId
ImportError: cannot import name 'EntityId' from 'tb_rest_client.models.models_ce' (/usr/local/lib/python3.7/dist-packages/tb_rest_client/models/models_ce/__init__.py)
```

The same issue persists regardless of whether tb_rest_client.rest_client_ce or tb_rest_client.rest_client_pe are imported.

**Comment by samson0v** [comment]
Hi @thunfischbrot, this bug was fixed, please update the rest-client via the master branch.

**Comment by zbeacon** [comment]
Hi @thunfischbrot ,

We have released the version 1.3.1 with fix for this point, please update your library and try the new version.
